marimo/_ast/pytest.py

```diff
@@ -14,10 +14,11 @@ def _pytest_scaffold(stub: Any) -> Any:
     return stub(stub=stub)
 
 
-PYTEST_BASE = ast.parse(inspect.getsource(_pytest_scaffold))
-
-
 def wrap_fn_for_pytest(func: Fn, cell: Cell) -> Callable[..., Any]:
+    # Avoid declaring the function in the global scope, since it may cause
+    # issues with meta-analysis tools like cxfreeze (see #3828).
+    PYTEST_BASE = ast.parse(inspect.getsource(_pytest_scaffold))
```

Comment:
anyway to add a test for this? maybe in CI something cheap like `uvx cxfreeze <some file>`

Reply:
Was thinking the same thing, but not replicating locally (maybe also because editable install). Will dig into it a bit more, since it seems like a common trap- but maybe not till tomorrow

Reply:
@dmadisetti want me to merge in the meantime so the next release picks it up?

Reply:
Sure! Created #3834 to track the issue